litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -307,11 +317,18 @@ func UpdateWorkflow(workflow *model.ChaosWorkFlowInput, r store.StateData) (*mod
 		return nil, err
 	}
 
+	workflowNamespace := gjson.Get(workflow.WorkflowManifest, "metadata.namespace").String()
```

Comment:
Is there a need to add a new package? json.Unmarshal would work too?

Reply:
WorkflowManifest is JSON string and to unMarshal WorkflowManifest we need a struct of workflow manifest

Reply:
We can use map[string]inteface{} to locate the "namespace" key, if it is there we use it or else we use the os.Getenv()

Reply:
Yes, we can use but just to ease the getting of the namespace. I have used it. 
We have used similar pkg for setting also

Reply:
FYI map[string]map[string]interface{} or map[string]interface{} is not able to parse the manifest

Reply:
Why is map[string]interface{} not able to parse it? I remember parsing it while I made the changes for the wf_id in cron-workflows?

Reply:
We can use this package, i think Ishan already used the sjson package to add the wf_id.

Reply:
You have use sjson to set wf_id in cron-workflow, For parsing this kind of JSON, it requires proper struct.

Reply:
No I did not use sjson, it was already there I would've used simple map for that also. 

This is the unmarshaling code I used 
```
var workflow map[string]interface{}
	err := json.Unmarshal([]byte(input.WorkflowManifest), &workflow)
	if err != nil {
		return nil, err
	}
```

Reply:
To get the ns you would have to marshal twice maybe, so it's easier to use this gjson package

Reply:
Okay, Is it good to merge then?